test/screenshot/tab-bar/index.js

```diff
@@ -9,6 +9,12 @@ class TabBarTest extends React.Component {
     activeIndex: this.props.activeIndex || 0, // eslint-disable-line react/prop-types
   };
 
+  componentDidUpdate(prevProps) {
```

Comment:
This can be removed if we don't wrap the test components in too many layers. See comments below

Reply:
Good call!